recipe/activate.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -ex
+
+[[ "@target_platform@" == "linux-64" ]] && targetsDir="targets/x86_64-linux"
+[[ "@target_platform@" == "linux-ppc64le" ]] && targetsDir="targets/ppc64le-linux"
+[[ "@target_platform@" == "linux-aarch64" ]] && targetsDir="targets/sbsa-linux"
```

Comment:
I'm observing that targetsDir is an empty string. This probably means that "@target_platform@" is undefined or doesn't equal any of these linux options. What should I be using instead?

Reply:
Probably `$target_platform`

Reply:
Yeah `$target_platform` should be fine here. The `@` syntax is used for substitution in the other feedstock https://github.com/conda-forge/cuda-nvcc-feedstock/blob/main/recipe/build.sh#L10

Reply:
Interesting. What is the utility of using a template like that then replacing it during install?